### ISSUE TITLE
eclipse fails to install with "missing filename"

### ISSUE THREAD

**Issue by romildo** [NixOS/nixpkgs#15079]
On my NixOS unstable system, when `eclipses.eclipse-platform` or `eclipses.eclipse-sdk` is added to `nixpkgs.config.packageOverrides` on the system configuration file, `nixos-rebuild` fails with the message
```
# nixos-rebuild switch
[...]
eclipse/readme/readme_eclipse.html
missing filename
builder for ‘/nix/store/i8lfzisbx4i6vq5z84gb9av337b3m2m0-eclipse-platform-4.5.2.drv’ failed with exit code 1
[...]
```

**Comment by dezgeg** [comment]
Maybe fixed by https://github.com/NixOS/nixpkgs/pull/15058

**Comment by romildo** [comment]
Yes, it seems to be fixed by #15058.

**Comment by dezgeg** [comment]
Thanks for testing, I merged it now.
